tests/utils.py

```diff
@@ -3,6 +3,12 @@
 from apispec.utils import build_reference
 
 
+def get_examples(spec):
+    if spec.openapi_version.major < 3:
```

Comment:
This condition should be removed.

Reply:
Fixed in 0776441f5060f10767cdd773fc5961c252a024c0